test-graalvm/oracle/build.gradle

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id 'io.micronaut.build.internal.r2dbc-testproject'
+}
+
+dependencies {
+    runtimeOnly(libs.managed.r2dbc.oracle)
+    runtimeOnly("com.oracle.database.jdbc:ojdbc11:21.8.0.0")
```

Comment:
can this version be in libs.versions.toml?

Reply:
After sql module is updated